.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+versions:
+  - "2.7"
+  - "3.5"
+cache: pip
+sudo: required
+services:
+  - docker
+
+install:
+  - pip install -U pip
+  - pip --version
+  - pip install ansible ansible-lint yamllint flake8 molecule docker-py
+  - ansible-lint --version
```

Comment:
We probably should also toss an `ansible --version` in here. I'll update the skeleton to include it.